dotnet/src/Experimental/Process.Runtime.Dapr/Actors/ProcessActor.cs

```diff
@@ -350,6 +353,41 @@ private async Task EnqueueExternalMessagesAsync()
         }
     }
 
+    /// <summary>
+    /// Check for the presence of an global-error event and any edges defined for processing it.
+    /// When both exist, the error event is processed and sent to the appropriate targets.
+    /// </summary>
+    private async Task HandleGlobalErrorMessageAsync()
+    {
+        var scopedEventBufferId = new ActorId(ProcessConstants.GlobalErrorEventId);
```

Comment:
Does this buffer Id need to be scoped to the process? As is, I believe it's global to the Dapr runtime right?

Reply:
Ah I see, yeah: `$"{globalid}{processid}"`.  Good catch

Reply:
Updated